package sv_cl_messages;

message NetTable
{
	enum Command
	{
		CLEAR = 0;
		UPDATE = 1;
	}
	
	required Command command = 1;
	
	oneof oneof_tableID
	{
		string name = 2; //Only used by CREATE.
		uint32 tableID = 3;
	}
	
	//Used by UPDATE.
	optional bytes data = 4;
}